apps/web/components/BlockRecommendedProducts/BlockRecommendedProducts.vue

```diff
@@ -39,149 +39,95 @@
       <div class="mb-6">
         <UiFormLabel class="mb-1">Text Color</UiFormLabel>
 
-        <SfInput v-model="recomandedBlockValue.text.color" type="text">
+        <SfInput v-model="recommendedBlockValue.text.color" type="text">
           <template #suffix>
             <label
               for="text-color"
-              :style="{ backgroundColor: recomandedBlockValue.text.color }"
+              :style="{ backgroundColor: recommendedBlockValue.text.color }"
               class="rounded-lg cursor-pointer"
             >
-              <input id="text-color" v-model="recomandedBlockValue.text.color" type="color" class="invisible w-8" />
+              <input id="text-color" v-model="recommendedBlockValue.text.color" type="color" class="invisible w-8" />
             </label>
           </template>
         </SfInput>
       </div>
       <div class="mb-6">
-        <UiFormLabel class="mb-1">Textbox Background</UiFormLabel>
+        <UiFormLabel class="mb-1">Text Alignment (y)</UiFormLabel>
```

Comment:
```suggestion
        <UiFormLabel class="mb-1">Text alignment</UiFormLabel>
```
**question**: Why "(y)"?